contributed_definitions/nyaml/NXcalibration.yaml

```diff
@@ -49,13 +60,38 @@ NXcalibration(NXobject):
     unit: NX_ANY
     doc: |
       A vector representing the axis after calibration, matching the data length
+    \@output_path:
+      doc: |
+        The path to which this data is written, e.g., the calibrated energy.
+        Should be a valid NeXus path name, e.g., /entry/data/energy.
     dimensions:
       rank: 1
       dim: [[1, ncal]]
   original_axis(NX_FLOAT):
     unit: NX_ANY
     doc: |
       Vector containing the data coordinates in the original uncalibrated axis
+    \@symbol:
+      doc: |
+        The symbol of the axis to be used in the fit_function, e.g., `energy`, `E`.
+        This should comply to python variable naming conventions.
```

Comment:
I know this is about naming conventions and not actual implementation in any language, but maybe it is anyway not the best idea to reference Python. You could just say "Naming convention: lowe case, snakecase".

Reply:
I added a list of naming conventions for python